sova/dashboard/services/pr_service.py

```
@@ -96,12 +98,18 @@ def _extract_linked_issue(raw: dict) -> int | None:
+    if m:
+        return int(m.group(1))
+    m = _TITLE_JIRA_KEY_RE.search(title)
     if m:
         return int(m.group(1))
     branch = raw.get("headRefName") or ""
     m = _BRANCH_ISSUE_RE.search(branch)
     if m:
         return int(m.group(1))
+    m = _BRANCH_JIRA_KEY_RE.search(branch)
+    if m:
+        return int(m.group(1))
```

Ruff (0.16.0)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)